subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/LintReportTask.kt

```diff
@@ -87,11 +105,12 @@ abstract class LintReportTask : DefaultTask(), BuildVerdictTask {
     private fun parseReports(): List<LintReportModel> = LintResultsParser(reportsDir, project.ciLogger).parse()
 
     private fun mergeReports(models: List<LintReportModel>): File {
-        val report = project.file(project.buildDir.path + "/reports/lint/lint-merge.html")
-        LintReportMerger(models, report).write()
+        val reportFile = project.file(project.buildDir.path + "/reports/lint/lint-merge.html")
```

Comment:
I thought that we don't use a merged report but only original one from an application module.